src/sentry/tasks/statistical_detectors.py

```diff
@@ -281,7 +275,11 @@ def detect_transaction_trends(
     #
     # If we filter this in the earlier step, it makes the initial dispatch
     # task take longer than necessary.
-    projects = Project.objects.filter(id__in=project_ids)
+    projects = [
+        project
+        for project in Project.objects.filter(id__in=project_ids).select_related("organization")
+        if features.has("organizations:performance-statistical-detectors-ema", project.organization)
+    ]
```

Comment:
I feel like we could use `values_list` similarly here because downstream I think we only pull off organization ID and project ID, but that would require more code changes. Maybe not necessary until we see the impacts of this.